packages/queries/src/getColumns.ts

```diff
@@ -0,0 +1,33 @@
+import { DbSchema } from './types/schemas';
+
+export function getColumns(schema: DbSchema, table: string) {
```

Comment:
With regards to the implementation: this can be cleaned up much more if you have the right building blocks:

Remember what I said about divide and conquer? Split the problem into 3-4 basic functions.

- getSelectableColumns(schema, table)
- getTableDef(schema, table)
- isSelectableColumn(columnDef):boolean

```ts
// pseudocode
getSelectableColumns(schema,table) {
    const table = getTableDef(schema,table);
    for (const column of getColumnDefs(table)) {
         if (isSelectableColumn(column)) {
              // TODO
         }
    }
}
```